configs/config-dev.json

```diff
@@ -15,6 +15,20 @@
       },
       "commonOptions":{
         "authServerDomain":"auth.test.com"
+      },
+      "keycloakOptions":{
+        "keycloakHost": "identitydev.iudx.io",
+        "keycloakPort": 8443,
+        "keycloakRealm": "some-realm",
+        "keycloakSite": "{{protocol}}://{{keycloakHost}}:{{keycloakPort}}/auth/realms/{{keycloakRealm}}",
+        "keycloakUrl": "{{protocol}}://{{keycloakHost}}:{{keycloakPort}}/auth",
+        "keycloakTokenUri": "/auth/realms/{{keycloakRealm}}/protocol/openid-connect/token",
```

Comment:
- Some of the keycloak options are going to be removed once the token revocation changes start, so I didn't want to change it now. I think the `keycloakTokenUri`, `keycloakHost` and `keycloakPort` fields will go
- Currently, the Admin and Registration verticles need the `keycloakUrl` in that format, while the ApiServer verticle needs the `keycloakSite` in that format. It's a little difficult to make it uniform here since they use different libs - the former uses the KeycloakAdmin library, while the latter uses the Vert.x OAuth libraries. ConfigResolve.resolve() can't resolve inside an options object also. I'll see what changes can be made to simplify this, but it may take some more work.